include/cuco/detail/open_addressing/open_addressing_impl.cuh

```diff
@@ -539,6 +540,50 @@ class open_addressing_impl {
     return counter.load_to_host(stream);
   }
 
+  // TODO docs
```

Comment:
TODO

Reply:
Addressed in [dcb0536](https://github.com/NVIDIA/cuCollections/pull/380/commits/dcb0536985e973ef9a377691f408603c8dfd0364)